vere/main.c

```diff
@@ -546,6 +553,66 @@ main(c3_i   argc,
     return 1;
   }
 
+  /*
+    In daemon mode, run the application as a background process.
+
+    - First setup a pipe and `fork()` the process.
+    - In the parent, read one byte from the pipe and then exit.
+    - In the child, write one byte to the pipe and then continue as per usual.
+
+    First run `fork()`,
+
+    TODO Redirect output to a file.
+    TODO Don't exit from the parent until the boot sequence finishes.
+    TODO Exit with an error code if the child process failed to boot.
+  */
+  if ( c3y == u3_Host.ops_u.dem ) {
+    int pipefd[2];
+    char buf[2] = {0,0};
+
+    if (0 != pipe(pipefd)) {
+      return 1; // TODO
+    }
+
+    fprintf(stderr, "forking\n");
+    fflush(stderr);
+    pid_t pid = fork();
+
+    if (pid) {
+      close(pipefd[1]);
+      close(0);
```

Comment:
I think we should keep the notation consistent and use `stdin` and `stdout` instead of `0` and `1`.